src/main/java/org/folio/circulation/domain/notice/TemplateContextUtil.java

```diff
@@ -108,6 +109,29 @@ public static JsonObject createStaffSlipContext(Request request) {
     return createStaffSlipContext(request.getItem(), request);
   }
 
+  public static JsonObject addPrimaryServicePointNameToStaffSlipContext(JsonObject entries, ServicePoint primaryServicePoint) {
+    if (primaryServicePoint == null) {
+      log.info("addPrimaryServicePointNameToStaffSlipContext:: parameters entries: {}, primaryServicePoint = null", entries);
+      return entries;
+    }
+
+    if (entries == null) {
+      log.info("addPrimaryServicePointNameToStaffSlipContext:: parameters entries: null, primaryServicePointName {}", primaryServicePoint.getName());
+      return new JsonObject();
+    }
+
+    log.debug("addPrimaryServicePointNameToStaffSlipContext:: parameters entries: {}, primaryServicePointName = {}", entries, primaryServicePoint.getName());
+
+    entries.getJsonArray(PICK_SLIPS_KEY)
+      .stream()
+      .map(JsonObject.class::cast)
+      .map(pickSlip -> pickSlip.getJsonObject(ITEM))
+      .forEach(item -> item.put("effectiveLocationPrimaryServicePointName", primaryServicePoint.getName()));
+
+    log.debug("addPrimaryServicePointNameToStaffSlipContext:: parameters entries: {}, primaryServicePointName = {}", entries, primaryServicePoint.getName());
```

Comment:
You don't need to log the parameters multiple times. Instead, please log the result using the following format:

LOGGER.info("sumWithCondition:: result: {}", sumResult);

return sumResult;

You can refer to the documentation for logging at https://wiki.folio.org/display/DD/Logging.